tools/build/suites/weekly/canary_list

```diff
@@ -0,0 +1,203 @@
+AIOLIST="webapi-noneservice-tests
```

Comment:
Do you have list like `beta/stable_list` in the master branch?

If not, why don't use the naming as https://github.com/crosswalk-project/crosswalk-test-suite/pull/2497#issuecomment-121534484

> `tools/build/suites/WEEKLY`, `tools/build/suites/WTS`